base/sysimg.jl

```diff
@@ -77,6 +77,7 @@ let
 
     tot_time_stdlib = 0.0
     # use a temp module to avoid leaving the type of this closure in Main
+    push!(empty!(LOAD_PATH), "@stdlib")
```

Comment:
Since we set this in sysimage.mk, how does it end up not being correct here already from there?

Reply:
I didn't know we set it in `sysimage.mk`, but It's not getting set [where it matters](https://github.com/JuliaLang/julia/blob/43a14f8401ffa2590328c44f65796d3e7b68263c/sysimage.mk#L68-L72).